protos/core.proto

```diff
@@ -142,6 +142,8 @@ message CompileConfig {
   // Deprecated.
   string schema_suffix_override = 2;
   string return_override = 5;
+
+  map<string, string> vars = 8;
```

Comment:
i don't understand why we need this... it's already inside `project_config_override`